wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -272,4 +276,10 @@ public void setScreenOrientation(ScreenOrientation orientation) throws Exception
     public void unsetScreenOrientation() throws Exception {
         getUiDevice().unfreezeRotation();
     }
+
+   public void uiDevicePerformLongClick(UiObject view, int steps) throws Exception {
```

Comment:
Couldn't the same be said of all the methods in the base class that use steps in that case. I think we would be setting a new precedent here.

Also the android api reference states:
> Steps are injected about 5 milliseconds apart, so 100 steps may take around 0.5 seconds to complete.

In general I think there needs to be a task/ticket to tidy up the base class/es since there a few things that are a bit woolly. i.e. variable names are of mixed case, the methods aren't const correct, instance methods on static objects etc. Other changes like default step values could be added then.